### PR TITLE
Pass along any Codecov argument

### PR DESCRIPTION
**Pull request by stevepeak**
Goal is to have customers include any number of additional arguments listed here: https://github.com/codecov/codecov-bash/blob/4c9c68c8aadf658812518a2b9a72a9038ee32dff/codecov#L53-L108

Thanks!

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -28,3 +28,10 @@ inputs:
       description: |
         Codecov.io repository upload token
       is_required: true
+  - codecov: "--codecov"
+    opts:
+      title: Additional options for Codecov call
+      description: |-
+        Options added to the end of the Gradle call.
```
Comment (viktorbenei) [inline step.yml]: I don't think this will be a Gradle call ;)

File: step.yml [hunk 2]
```diff
@@ -28,3 +28,11 @@ inputs:
...
         Codecov.io repository upload token
       is_required: true
+  - codecov: "--codecov"
+    opts:
+      title: Additional options for Codecov call
+      description: |-
+        Options to pass along to the Codecov uploader.
+        You can use multiple options, separated by a space.
+        Review all options at https://github.com/codecov/codecov-bash
+        character. Example: `--codecov -F unittests -J '^Project'`
```
Comment (godrei) [inline step.yml]: Hi, are you sure about `Review all options at https://github.com/codecov/codecov-bash __character__`?

Comment (stevepeak) [inline step.yml]: woops. My proofreading is poor. Thanks!

**Comment by viktorbenei** [comment]
LGTM 👍

**Comment by stevepeak** [comment]
@viktorbenei @godrei thank you both for very quick help and support. I really appreciate it 👍

**Pull request merged**